cluster/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/cluster/ClusterComponents.scala

```diff
@@ -6,12 +6,11 @@ package com.lightbend.lagom.scaladsl.cluster
 import akka.actor.ActorSystem
 import akka.cluster.Cluster
 import com.lightbend.lagom.internal.cluster.JoinClusterImpl
-import com.lightbend.lagom.scaladsl.playjson.RequiresJsonSerializerRegistry
 
 /**
  * Cluster components (for compile-time injection).
  */
-trait ClusterComponents extends RequiresJsonSerializerRegistry {
+trait ClusterComponents {
```

Comment:
Should we not require users to provide a `SerializerRegistry` instead? 

I know, we don't have such a thing it, but how can we enforce it now?

Reply:
Agree, I think `RequiresJsonSerializerRegistry` should become `RequiresSerializerRegistry`.

Reply:
Why? This is what I was trying to say in my [comment on #954](https://github.com/lagom/lagom/issues/954#issuecomment-339909281). It doesn't seem to need such a thing.